<ul>
  
  <li>
    <span class="authors"> David Criens, Peter Pfaffelhuber, and Thorsten Schmidt. </span>
    <span class="title">
       
      <a href="https://doi:10.1214/23-EJP902">The martingale problem method revisited</a>. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Electronic Journal of Probability</span>,
    <span class="date">
       2023.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Peter Pfaffelhuber and Anton Wakolbinger. </span>
    <span class="title">
       
      <a href="https://doi:10.1214/23-EJP934">A diploid population model for copy number variation of genetic elements</a>. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Electronic Journal of Probability</span>,
    <span class="date">
       2023.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Timo Enger and Peter Pfaffelhuber. </span>
    <span class="title">
        A unified framework for limit results in chemical reaction networks on multiple time-scales. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Electronic Journal of Probability</span>,
    <span class="date">
       2023.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Peter Pfaffelhuber, Stefan Rotter, and Jakob Stiefel. </span>
    <span class="title">
       
      <a href="https://doi.org/10.1016/j.spa.2022.07.006">Mean-field limits for non-linear Hawkes processes with excitation and inhibition</a>. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Stochastic Processes and their Applications</span>,
    <span class="date">
       2022.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Peter Pfaffelhuber, Elisabeth Sester-Huss, Franz Baumdicker, Jana Naue, Sabine Lutz-Bonengel, and Fabian Staubach. </span>
    <span class="title">
        Inference of recent admixture using genotype data. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Forensic Science International: Genetics</span>,
    <span class="date">
       2022.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Klara Elisabeth Burger, Peter Pfaffelhuber, and Franz Baumdicker. </span>
    <span class="title">
       
      <a href="https://doi.org/10.1371/journal.pcbi.1010407">Neural networks for self-adjusting mutation rate estimation when the recombination rate is unknown</a>. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">PLOS Computational Biology 18(8)</span>,
    <span class="date">
       2022.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Peter Pfaffelhuber and Angelika Rohde. </span>
    <span class="title">
       
      <a href="https://doi.org/10.1016/j.tpb.2022.09.003">A central limit theorem concerning uncertainty in estimates of individual admixture</a>. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Theoretical Population Biology</span>,
    <span class="date">
       2022.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Veronika Lipphardt, Mihai Surdu, Nils Ellebrecht, Peter Pfaffelhuber, Matthias Wienroth, and Gudrun A Rappold. </span>
    <span class="title">
       
      <a href="https://doi.org/10.1038/d41586-021-03416-3">Europe’s Roma people are vulnerable to poor practice in genetics</a>. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Nature 599(7885)</span>,
    <span class="date">
       2021.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Peter Pfaffelhuber and Jakob Stiefel. </span>
    <span class="title">
        The range of once-reinforced random walk in one dimension. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Random Structures \& Algorithms 58(1)</span>,
    <span class="date">
       2021.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Felix Hermann and Peter Pfaffelhuber. </span>
    <span class="title">
        The partial duplication random graph with edge deletion. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">ALEA</span>,
    <span class="date">
       2021.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Nora Scherer, Peggy Sekula, Peter Pfaffelhuber, Anna Koettgen, and Pascal Schlosser. </span>
    <span class="title">
        Pgainsim: A Method to Assess the Mode of Inheritance for Quantitative Trait Loci in Genome-Wide Association Studies. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">HUMAN HEREDITY</span>,
    <span class="date">
       2021.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Peter Pfaffelhuber, Franziska Grundner-Culemann, Veronika Lipphardt, and Franz Baumdicker. </span>
    <span class="title">
       
      <a href="https://doi:10.1016/j.fsigen.2020.102259">How to choose sets of ancestry informative markers: A supervised feature selection approach</a>. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Forensic Science International: Genetics</span>,
    <span class="date">
       2020.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Michael Lynch, Bernhard Haubold, Peter Pfaffelhuber, and Takahiro Maruki. </span>
    <span class="title">
        Inference of historical population-size changes with allele-frequency data. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">G3: Genes, Genomes, Genetics 10(1)</span>,
    <span class="date">
       2020.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Felix Hermann and Peter Pfaffelhuber. </span>
    <span class="title">
       
      <a href="https://doi.org/10.1016/j.spa.2019.07.011">Markov branching processes with disasters: extinction, survival and duality to p-jump processes</a>. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Stochastic Processes and their Applications 130(4)</span>,
    <span class="date">
       2020.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Franz Baumdicker, Elisabeth Sester-Huss, and Peter Pfaffelhuber. </span>
    <span class="title">
        Modifiers of mutation rate in selectively fluctuating environments. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Stochastic Processes and their Applications 130(11)</span>,
    <span class="date">
       2020.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Elisabeth Huss and Peter Pfaffelhuber. </span>
    <span class="title">
       
      <a href="https://doi.org/10.1016/j.tpb.2019.10.002">Genealogical distances under low levels of selection</a>. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Theoretical Population Biology</span>,
    <span class="date">
       2020.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Veronika Lipphardt, Nicholas Buchanan, Peter Pfaffelhuber, Fabian Staubach, and Matthias Wienroth. </span>
    <span class="title">
       
      <a href="https://doi.org/10.1515/jwiet-2019-0006">Interdisziplin\"are Überlegungen zu Erweiterten DNA-Analysen</a>. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Jahrbuch f\"ur Wissenschaft und Ethik 24(1)</span>,
    <span class="date">
       2019.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Anton Pirogov, Peter Pfaffelhuber, Angelika B{\"o}rsch-Haubold, and Bernhard Haubold. </span>
    <span class="title">
        High-complexity regions in mammalian genomes are enriched for developmental genes. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Bioinformatics 35(11)</span>,
    <span class="date">
       2019.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Nicholas Buchanan, Fabian Staubach, Matthias Wienroth, Peter Pfaffelhuber, Mihai Surdu, Anna Lipphardt, Anna K{\"o}ttgen, Veronika Lipphardt, and others. </span>
    <span class="title">
       
      <a href="https://doi: 10.1016/j.fsigen.2018.01.009">Forensic DNA phenotyping legislation cannot be based on “Ideal FDP”—A response to Caliebe, Krawczak and Kayser (2017)</a>. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Forensic Science International: Genetics</span>,
    <span class="date">
       2018.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Peter Czuppon and Peter Pfaffelhuber. </span>
    <span class="title">
        Limits of noise for autoregulated gene expression. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Journal of mathematical biology 77(4)</span>,
    <span class="date">
       2018.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Sebastian Bossert and Peter Pfaffelhuber. </span>
    <span class="title">
        The fixation probability and time for a doubly beneficial mutant. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Stochastic Processes and their Applications 128(12)</span>,
    <span class="date">
       2018.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Franz Baumdicker, Ariana MI Huebner, and Peter Pfaffelhuber. </span>
    <span class="title">
       
      <a href="https://doi: 10.1016/j.tpb.2017.11.001">The independent loss model with ordered insertions for the evolution of CRISPR spacers</a>. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Theoretical Population Biology</span>,
    <span class="date">
       2018.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> P Pfaffelhuber and A Wakolbinger. </span>
    <span class="title">
        Fixation probabilities and hitting times for low levels of frequency-dependent selection. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Theoretical Population Biology</span>,
    <span class="date">
       2018.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Peter Czuppon and Peter Pfaffelhuber. </span>
    <span class="title">
        A spatial model for selection and cooperation. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Journal of Applied Probability 54(2)</span>,
    <span class="date">
       2017.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Felix Hermann and Peter Pfaffelhuber. </span>
    <span class="title">
        Large-scale behavior of the partial duplication random graph.. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">ALEA. Latin American Journal of Probability \& Mathematical Statistics</span>,
    <span class="date">
       2016.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Andreas Greven, Peter Pfaffelhuber, Cornelia Pokalyuk, and Anton Wakolbinger. </span>
    <span class="title">
        The fixation time of a strongly beneficial allele in a structured population. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Electron. J. Probab 21(61)</span>,
    <span class="date">
       2016.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Martin Jansen, Peter Pfaffelhuber, Michael M Hoffmann, Gerhard Puetz, and Karl Winkler. </span>
    <span class="title">
        In silico modeling of the dynamics of low density lipoprotein composition via a single plasma sample [S]. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Journal of lipid research 57(5)</span>,
    <span class="date">
       2016.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Bernhard Haubold, Fabian Kl{\"o}tzl, and Peter Pfaffelhuber. </span>
    <span class="title">
        andi: Fast and accurate estimation of evolutionary distances between closely related genomes. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Bioinformatics 31(8)</span>,
    <span class="date">
       2015.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Jan-Philip Schl{\"u}ter, Peter Czuppon, Oliver Schauer, Peter Pfaffelhuber, Matthew McIntosh, and Anke Becker. </span>
    <span class="title">
        Classification of phenotypic subpopulations in isogenic bacterial cultures by triple promoter probing at single cell level. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Journal of Biotechnology</span>,
    <span class="date">
       2015.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Martin Jansen and Peter Pfaffelhuber. </span>
    <span class="title">
        Stochastic gene expression with delay. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Journal of theoretical biology</span>,
    <span class="date">
       2015.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> PETER PFAFFELHUBER and LEA POPOVIC. </span>
    <span class="title">
        SCALING LIMITS OF SPATIAL COMPARTMENT MODELS FOR CHEMICAL REACTION NETWORKS. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">The Annals of Applied Probability 25(6)</span>,
    <span class="date">
       2015.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Bence M{\'e}lyk{\'u}ti and Peter Pfaffelhuber. </span>
    <span class="title">
        The stationary distribution of a markov jump process glued together from two state spaces at two vertices. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Stochastic Models 31(4)</span>,
    <span class="date">
       2015.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Peter Pfaffelhuber and Lea Popovic. </span>
    <span class="title">
        How spatial heterogeneity shapes multiscale biochemical reaction network dynamics. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Journal of the Royal Society Interface 12(104)</span>,
    <span class="date">
       2015.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Andrej Depperschmidt, {\'E}tienne Pardoux, and Peter Pfaffelhuber. </span>
    <span class="title">
        A mixing tree-valued process arising under neutral evolution with recombination. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Electronic Journal of Probability</span>,
    <span class="date">
       2015.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Michael Lynch, Sen Xu, Takahiro Maruki, Xiaoqian Jiang, Peter Pfaffelhuber, and Bernhard Haubold. </span>
    <span class="title">
        Genome-wide linkage-disequilibrium profiles from single individuals. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Genetics 198(1)</span>,
    <span class="date">
       2014.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Franz Baumdicker and Peter Pfaffelhuber. </span>
    <span class="title">
        The infinitely many genes model with horizontal gene transfer. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Electron. J. Probab 19(115)</span>,
    <span class="date">
       2014.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> A. Depperschmidt, P. Pfaffelhuber, and A. Scheuringer. </span>
    <span class="title">
        Some large deviations in Kingman's coalescent. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Elec. Comm. Probab. 20(7)</span>,
    <span class="date">
       2014.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> P. Czuppon and P. Pfaffelhuber. </span>
    <span class="title">
        Some limit results for Markov chains indexed by trees. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Elect. Comm. in Probab. 19(77)</span>,
    <span class="date">
       2014.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Sebastian Bossert and Peter Pfaffelhuber. </span>
    <span class="title">
        Correction: The Yule Approximation for the Site Frequency Spectrum after a Selective Sweep. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Plos one 9(1)</span>,
    <span class="date">
       2014.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Andreas Greven, Peter Pfaffelhuber, and Anita Winter. </span>
    <span class="title">
        Tree-valued resampling dynamics martingale problems and applications. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Probability Theory and Related Fields 155(3-4)</span>,
    <span class="date">
       2013.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Cornelia Pokalyuk and Peter Pfaffelhuber. </span>
    <span class="title">
        The ancestral selection graph under strong directional selection. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Theoretical population biology</span>,
    <span class="date">
       2013.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Bernhard Haubold, Linda Krause, Thomas Horn, and Peter Pfaffelhuber. </span>
    <span class="title">
        An alignment-free test for recombination. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Bioinformatics 29(24)</span>,
    <span class="date">
       2013.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Andrej Depperschmidt, Andreas Greven, and Peter Pfaffelhuber. </span>
    <span class="title">
        Path-properties of the tree-valued Fleming--Viot process. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Electron. J. Probab 18(84)</span>,
    <span class="date">
       2013.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> A Depperschmidt, N Ketterer, and P Pfaffelhuber. </span>
    <span class="title">
        A Brownian ratchet for protein translocation including dissociation of ratcheting sites. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Journal of mathematical biology 66(3)</span>,
    <span class="date">
       2013.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Cornelia Pokalyuk, Lisha A Mathew, Dirk Metzler, and Peter Pfaffelhuber. </span>
    <span class="title">
        Competing islands limit the rate of adaptation in structured populations. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Theoretical population biology</span>,
    <span class="date">
       2013.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> M McIntosh, P Czuppon, K Best, A Becker, and P Pfaffelhuber. </span>
    <span class="title">
        Modeling quorum sensing in Sinorhizobium meliloti. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">International Journal of Biomathematics and Biostatistics</span>,
    <span class="date">
       2013.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Sebastian Bossert and Peter Pfaffelhuber. </span>
    <span class="title">
        The yule approximation for the site frequency spectrum after a selective sweep. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">PLoS One 8(12)</span>,
    <span class="date">
       2013.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Franz Baumdicker, Wolfgang R Hess, and Peter Pfaffelhuber. </span>
    <span class="title">
        The infinitely many genes model for the distributed genome of bacteria. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Genome biology and evolution 4(4)</span>,
    <span class="date">
       2012.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Andrej Depperschmidt, Andreas Greven, and Peter Pfaffelhuber. </span>
    <span class="title">
       
      <a href="https://doi:10.1214/11-AAP831">Tree-valued Fleming-Viot dynamics with mutation and selection</a>. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Annals of applied probability 22(6)</span>,
    <span class="date">
       2012.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> P Pfaffelhuber, PR Staab, and A Wakolbinger. </span>
    <span class="title">
        Muller's ratchet with compensatory mutations.. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Annals of applied probability 22(5)</span>,
    <span class="date">
       2012.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Bernhard Haubold and Peter Pfaffelhuber. </span>
    <span class="title">
        Alignment-free population genomics: an efficient estimator of sequence diversity. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">G3: Genes| Genomes| Genetics 2(8)</span>,
    <span class="date">
       2012.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Holger F Biehler and Peter Pfaffelhuber. </span>
    <span class="title">
        Compact metric measure spaces and $\Lambda$-coalescents coming down from infinity. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">ALEA</span>,
    <span class="date">
       2012.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> A. Depperschmidt, A. Greven, and P. Pfaffelhuber. </span>
    <span class="title">
        Marked metric measure spaces. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Elect. Comm. in Probab.</span>,
    <span class="date">
       2011.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Peter Pfaffelhuber, Anton Wakolbinger, and Heinz Weisshaupt. </span>
    <span class="title">
        The tree length of an evolving coalescent. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Probability theory and related fields</span>,
    <span class="date">
       2011.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Bernhard Haubold, Floyd A Reed, and Peter Pfaffelhuber. </span>
    <span class="title">
        Alignment-free estimation of nucleotide diversity. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Bioinformatics 27(4)</span>,
    <span class="date">
       2011.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Greg Ewing, Joachim Hermisson, Peter Pfaffelhuber, and Johannes Rudolf. </span>
    <span class="title">
        Selective sweeps for recessive alleles and for other modes of dominance. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Journal of mathematical biology 63(3)</span>,
    <span class="date">
       2011.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Aur{\'e}lien Tellier, Peter Pfaffelhuber, Bernhard Haubold, Lisha Naduvilezhath, Laura E Rose, Thomas St{\"a}dler, Wolfgang Stephan, and Dirk Metzler. </span>
    <span class="title">
        Estimating parameters of speciation models based on refined summaries of the joint site-frequency spectrum. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">PLoS One 6(5)</span>,
    <span class="date">
       2011.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> M Jansen, K Winkler, G Puetz, P Pfaffelhuber, and M Mann. </span>
    <span class="title">
        In silico modelling of human lipoprotein metabolism. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Atherosclerosis Supplements</span>,
    <span class="date">
       2011.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> P. Pfaffelhuber and H. Weisshaupt. </span>
    <span class="title">
        Sensitivity analysis of one parameter semigroups exemplified by the Wright--Fisher diffusion. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Intern. J. Funct. Anal., Oper. Th. Appl. 3(2)</span>,
    <span class="date">
       2011.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Bernhard Haubold, Peter Pfaffelhuber, and Michael Lynch. </span>
    <span class="title">
        mlRho--a program for estimating the population mutation and recombination rates from shotgun-sequenced diploid genomes. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Molecular ecology</span>,
    <span class="date">
       2010.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> F Baumdicker, WR Hess, and P Pfaffelhuber. </span>
    <span class="title">
        THE DIVERSITY OF A DISTRIBUTED GENOME IN BACTERIAL POPULATIONS. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">The Annals of Applied Probability</span>,
    <span class="date">
       2010.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> K. Best and P. Pfaffelhuber. </span>
    <span class="title">
        The Aldous--Shields model revisited (with application to cellular ageing). 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Elec. Comm. Probab.</span>,
    <span class="date">
       2010.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Andrej Depperschmidt and Peter Pfaffelhuber. </span>
    <span class="title">
        Asymptotics of a Brownian ratchet for protein translocation. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Stochastic processes and their applications 120(6)</span>,
    <span class="date">
       2010.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Bernhard Haubold and Peter Pfaffelhuber. </span>
    <span class="title">
        Experimentelle und theoretische Biologie: getrennte Welten?. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Biologie in unserer Zeit 40(1)</span>,
    <span class="date">
       2010.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Thomas Städler, Bernhard Haubold, Carlos Merino, Wolfgang Stephan, and Peter Pfaffelhuber. </span>
    <span class="title">
       
      <a href="https://academic.oup.com/genetics/article/182/1/205/6062953?login=true">The impact of sampling schemes on the site frequency spectrum in nonequilibrium subdivided populations</a>. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Genetics 182(1)</span>,
    <span class="date">
       2009.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Andreas Greven, Peter Pfaffelhuber, and Anita Winter. </span>
    <span class="title">
        Convergence in distribution of random metric measure spaces ($\Lambda$-coalescent measure trees). 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Probability Theory and Related Fields 145(1-2)</span>,
    <span class="date">
       2009.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Bernhard Haubold, Peter Pfaffelhuber, Mirjana Domazet-Los˘ o, and Thomas Wiehe. </span>
    <span class="title">
        Estimating mutation distances from unaligned genomes. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Journal of Computational Biology 16(10)</span>,
    <span class="date">
       2009.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Alison Etheridge, Peter Pfaffelhuber, and Anton Wakolbinger. </span>
    <span class="title">
        How often does the ratchet click? Facts, heuristics, asymptotics. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Trends in stochastic analysis</span>,
    <span class="date">
       2009.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> P Pfaffelhuber, A Lehnert, and W Stephan. </span>
    <span class="title">
        Linkage disequilibrium under genetic hitchhiking in finite populations. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Genetics 179(1)</span>,
    <span class="date">
       2008.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> J. Hermisson and P. Pfaffelhuber. </span>
    <span class="title">
        The pattern of genetic hitchhiking under recurrent mutation. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Elec. J. Probab. 13(68)</span>,
    <span class="date">
       2008.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> P Pfaffelhuber and A Studeny. </span>
    <span class="title">
        Approximating genealogies for partially linked neutral loci under a selective sweep. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Journal of mathematical biology 55(3)</span>,
    <span class="date">
       2007.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> A Etheridge, P Pfaffelhuber, and A Wakolbinger. </span>
    <span class="title">
        An approximate sampling formula under genetic hitchhiking. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">ANNALS OF APPLIED PROBABILITY 16(2)</span>,
    <span class="date">
       2006.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Peter Pfaffelhuber and Anton Wakolbinger. </span>
    <span class="title">
        The process of most recent common ancestors in an evolving coalescent. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Stochastic Processes and their Applications 116(12)</span>,
    <span class="date">
       2006.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Peter Pfaffelhuber, B Haubold, and A Wakolbinger. </span>
    <span class="title">
        Approximate genealogies under genetic hitchhiking. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Genetics 174(4)</span>,
    <span class="date">
       2006.
    </span>

    <span class="links">
      
    </span>
  </li>

  
  <li>
    <span class="authors"> Peter Pfaffelhuber and P Pfaffelhuber. </span>
    <span class="title">
        The Finite System Scheme for State-dependent interacting multitype Branching Systems. 
    </span>
    <span class="venuetype"></span
    ><span class="venue">Electronic Journal of Probability</span>,
    <span class="date">
       2003.
    </span>

    <span class="links">
      
    </span>
  </li>

  
</ul>
